source/docs/articles/wordpress/site-networks/getting-started.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Up and Running with WordPress Site Networks
+description: Get started with WordPress Site Networks. Learn about the Pantheon WordPress Site Network support, start developing, or import existing networks.
+---
+
+Once a Pantheon employee has given you access to your WordPress Site Network site, you can import an existing network or start from scratch and begin developing your project on the platform.
+
+## Migrating Site Networks to Pantheon
+
+If your network already exists, follow the instructions in the [Migrate to Pantheon: WordPress Site Networks](/docs/articles/sites/migrate/wordpress-site-networks/) doc.
+
+## Create a Network
+
+Before you begin, switch the Development environment to SFTP mode. Creating the network requires making modifications to the `wp-config.php` file using either WP-CLI or the WordPress Dashboard, which is not possible if the site's connection mode is set to Git.
+
+### Create a Network with WP-CLI (Recommended)
+
+If you’re comfortable with the command line, Terminus and WP-CLI let you convert an existing WordPress environment to Multisite in one command: `wp core multisite-convert`
+
+Before you begin, make sure that:
+
+- Your WordPress environment is functioning as expected.
+
+- [Terminus](/docs/articles/local/cli) is configured locally, and you’re authenticated with Pantheon.
+
+Once you’ve confirmed these prerequisites, you can enable WordPress Multisite with one command.
+
+Subdirectory style:
+```bash
+terminus wp core multisite-convert --site=<pantheon-site> --env=dev
+```
+
+Subdomains style:
+```bash
+terminus wp core multisite-convert --site=<pantheon-site> --env=dev --subdomains
+```
+
+Optionally specify --title to use a custom title for your Multisite Network:
+
+```bash
+terminus wp core multisite-convert --site=<pantheon-site> --env=dev --title=”My Awesome Multisite Network”
+```
+### Modify `DOMAIN_CURRENT_SITE`
+
+For compatibility with Pantheon, you’ll need to update `DOMAIN_CURRENT_SITE` to be set conditionally based on environment. Here is an example:
+
+<script src="https://gist.github.com/danielbachhuber/69c44664d4d63a6e19db.js"></script>
```

Comment:
Yes. I discovered how to do that yesterday while scripting `wp search-replace` operations for subdomain-style networks. https://gist-it.appspot.com/ is the secret sauce for organization in-repo gist embeds.  @rachelwhitton @joshkoenig do you have a preference for snippet storage? We currently have this docs repo, the https://github.com/pantheon-systems/pantheon-settings-examples repo, and the empty https://github.com/pantheon-systems/scripts repo to choose from. I'd like to add a set of `.sh` scripts into the scripts repo and embedding them in a doc: `wordpress/site-networks/cli-commands/`. large PR coming shortly.